doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1724,21 +1638,19 @@ String notations
 Tactic Notations
 -----------------
 
-Tactic notations allow to customize the syntax of tactics. They have the following syntax:
+Tactic notations allow customizing the syntax of tactics.
+
+.. cmd:: Tactic Notation {? ( at level @num ) } {+ @ltac_production_item } := @ltac_expr
 
-.. productionlist:: coq
-   tacn                 : Tactic Notation [`tactic_level`] [`prod_item` … `prod_item`] := `tactic`.
-   prod_item            : `string` | `tactic_argument_type`(`ident`)
-   tactic_level         : (at level `num`)
-   tactic_argument_type : ident | simple_intropattern | reference
-                        : hyp | hyp_list | ne_hyp_list
-                        : constr | uconstr | constr_list | ne_constr_list
-                        : integer | integer_list | ne_integer_list
-                        : int_or_var | int_or_var_list | ne_int_or_var_list
-                        : tactic | tactic0 | tactic1 | tactic2 | tactic3
-                        : tactic4 | tactic5
+   .. insertprodn ltac_production_item ltac_production_item
 
-.. cmd:: Tactic Notation {? (at level @num)} {+ @prod_item} := @tactic.
+   .. prodn::
+      ltac_production_item ::= @string
+      | @ident ( @ident {? , @string } )
+      | @ident
```

Comment:
> (`Tactic Notation foo := idtac.` fails with `Unknown entry foo.` This could be documented.)

Meaning it should be `Tactic Notation "foo" := idtac.`??  You think that's worth mentioning?

Reply:
Yes to your first question. Not sure to your second.